app/config/github.yml

```diff
@@ -23,12 +28,14 @@ parameters:
                 - app.subscriber.status_change_by_comment_subscriber
                 - app.subscriber.needs_review_new_pr_subscriber
                 - app.subscriber.bug_label_new_issue_subscriber
+                - app.subscriber.status_change_on_push_subscriber
```

Comment:
I think we should just enable this on the docs for now to see how it works. We would need to get the core team to agree to this, and I really want to keep Carson from doing anything "annoying". On the docs, we're better guinea pigs :)